docs/how_to_guide/plot_02_glm_demo.md

```diff
@@ -104,53 +127,61 @@
 
 print("Regularizer type:      ", type(model.regularizer))
 print("Observation model:", type(model.observation_model))
+```
+
+Hyperparameters can be set at any moment via the [`set_params`](nemos.glm.GLM.set_params) method.
 
-# %%
-# Hyperparameters can be set at any moment via the `set_params` method.
 
+```{code-cell} ipython3
```

Comment:
This code cell additionally prints a UserWarning for regularizer strength. Should it be suppressed?